openbsd/OpenBSDProcessList.c

```diff
@@ -76,6 +86,17 @@ ProcessList* ProcessList_new(UsersTable* usersTable, Hashtable* pidMatchList, ui
 
    opl->cpuSpeed = -1;
 
+   size = sizeof(cpu_stats);
+   for (unsigned int i = 0; i < ncpu; i++) {
+      ncmib[2] = i;
+      sysctl(ncmib, 3, &cpu_stats, &size, NULL, 0);
+      if (cpu_stats.cs_flags & CPUSTATS_ONLINE)
+         cpu_index_c++;
+
+      if (cpu_index_c == pl->cpuCount)
+         break;
+   }
```

Comment:
What is this loop about?
`cpu_index_c` is being increased and the loop is aborted when it reaches a specific value, but there are no further implications.

Reply:
Hmm, looks like I missed a bit when I was merging the patch which was originally done against 3.0.1. It will work on Intel CPUs without that loop (the skipped cpus are the higher numbered ones) but they're interspersed on AMD (skipped would be 1,3,5,7,..) I just pushed an updated version that I think should do the trick

Reply:
I asked someone to test on AMD, this version is looking ok.